docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,261 @@ Documentation for other releases can be found at
+`stop`		| `kubectl stop (-f FILENAME | TYPE (NAME | -l label | --all))` | Deprecated: Gracefully shut down a resource by name or filename
+`version`		| `kubectl version` | Display the Kubernetes version running on the client and server.
+
+Remember: For more about command operations, see the [kubectl](kubectl/kubectl.md) reference documentation.
+
+## Resource types
+
+The following table includes a list of all the supported resource types and their abbreviated aliases:
+
+Resource type	| Abbreviated alias
+-------------------- | --------------------
+`componentstatuses`	|	`cs`
+`events`	|	`ev`
+`endpoints`	|	`ep`
+`horizontalpodautoscalers`	|	`hpa`
+`limitranges`	|	`limits`
+`nodes`	|	`no`
+`namespaces`	|	`ns`
+`pods`	|	`po`
+`persistentvolumes`	|	`pv`
+`persistentvolumeclaims`	|	`pvc`
+`resourcequotas`	|	`quota`
+`replicationcontrollers`	|	`rc`
+`daemonsets`	|	`ds`
+`services`	|	`svc`
+`ingress`		|	`ing`
+
+## Output options
 
-This overview is intended for anyone who wants to use `kubectl` command line tool to interact with Kubernetes cluster. Please remember that it is built for quick started with `kubectl`; for complete and detailed information, please refer to [kubectl](kubectl/kubectl.md).
+Use the following sections for information about how you can format or sort the output of certain commands. For details about which commands support the various output options, see the [kubectl](kubectl/kubectl.md) reference documentation.
 
-TODO: auto-generate this file to stay up with `kubectl` changes. Please see [#14177](https://github.com/kubernetes/kubernetes/pull/14177).
+### Formatting output
 
-## Overview
+The default output format for all `kubectl` commands is the human readable ps output format. To output details to your terminal window in a specific format, you can add either the `-o` or `-output` flags to a supported `kubectl` command.
 
-`kubectl` controls the Kubernetes cluster manager. The synopsis is:
+#### Syntax
 
 ```
-kubectl [command] [TYPE] [NAME] [flags]
+kubectl [command] [TYPE] [NAME] -o=<output_format>
 ```
 
-This specifies:
+Depending on the `kubectl` operation, the following output formats are supported:
+
+Output format | Description
+--------------| -----------
+`-o=custom-columns=<spec>` | Print a table using a comma separated list of [custom columns](#custom-columns).
+`-o=custom-columns-file=<filename>` | Print a table using the [custom columns](#custom-columns) template in the `<filename>` file.
+`-o=json`     | Output a JSON formatted API object.
+`-o=jsonpath=<template>` | Print the fields defined in a [jsonpath](jsonpath.md) expression.
+`-o=jsonpath-file=<filename>` | Print the fields defined by the [jsonpath](jsonpath.md) expression in the `<filename>` file.
+`-o=name`     | Print only the resource name and nothing else. Supported only by [`kubectl label`](kubectl/kubectl_label.md).
+`-o=wide`     | Output in the default ps output format but include additional information like the node name.
```

Comment:
I got ps from all the examples below (ie. "// List all pods in ps output format.").

The Linux ps (process status) command, where: ps output = stdout = plain text = human readable. I think its probably clear to describe it so replace all "ps" with "plain-text" or "human readable".

Also, since `-o=wide` is only useful for pods, Ill add that to clarify.